### PR TITLE
HOV-44 | Select component - MultiSelect

### PR DESCRIPTION
**Pull request by saurabhsutar192**
MultiSelect enhancement to Select component

## Description
Added MultiSelect enhancement to Select component

## Motivation and Context
Provides functionality to select multiple items

## How Has This Been Tested?
local Build successful and manual testing

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

File: lib/src/components/Select/select.css.ts [hunk 1]
```diff
@@ -153,12 +151,27 @@ export const inputRecipe = recipe({
...
 export const inputTextContainer = style({
-  width: "85%",
+  width: "80%",
   fontSize: "16px",
   whiteSpace: "nowrap",
   overflow: "hidden",
+  height: "100%",
+});
+
+globalStyle(`${selectContainerStyles} [data-hover="true"]`, {
```
Comment (pruthvi2103) [inline lib/src/components/Select/select.css.ts:173]: Move this to a separate file called .glibalstyles.css.ts

**Review by pruthvi2103 — CHANGES_REQUESTED** [review]
One comment

File: lib/src/components/Select/select.css.ts (hunk 1 above)
Comment (saurabhsutar192) [reply lib/src/components/Select/select.css.ts:173]: sure

**Review by pruthvi2103 — APPROVED** [review]
LGTM

**Pull request merged**